frontend/src/pages/AccountSend.vue

```diff
@@ -743,8 +748,13 @@ function useSendForm() {
     if (amount) humanAmount.value = amount;
 
     // For token, we always default to the chain's native token if none was selected
-    if (paymentToken?.symbol) token.value = paymentToken;
-    else token.value = tokenList.value[0];
+    if (paymentToken?.symbol) {
+      token.value = paymentToken;
+      nativeToken.value = paymentToken;
```

Comment:
I don't understand the code changes on in this PR:
- `NATIVE_TOKEN` is a constant, defined per-chain
- `nativeToken, based on the variable name, sounds like it would be identical
- But it's not because if we have a payment link we set native token to the token in the payment link?
- So now we have:
    - `NATIVE_TOKEN` which is always the native token
    - `token` which is the selected token
    - `nativeToken` which is also the selected token?

Reply:
Possibly related, when I connect to sepolia and directly load http://localhost:8080/send I see an error

<img width="774" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/9b29abc3-2c82-4beb-82e4-54cdebb1f9f6">

Reply:
- So this was my work around to get `NATIVE_TOKEN` as `TokenInfoExtedned`, which directly casting it using `as` didn't work, I created `nativeToken` to be the `TokenInfoExtended` native token. 
- Payment link looks like a bug created from trying to use the same `setPaymentLinkData` function to set the default token.

Reply:
would just doing `NATIVE_TOKEN.value as unknown as TokenInfoExtended` fix the issue without needing a new variable? if so, you should add a comment about why the intermediate cast to `unknown` is safe, presumably here it's because we don't need the `minSendAmount` field of `TokenInfoExtended` and we just need the type as `TokenInfoExtended` for reason XYZ

Reply:
Fixed the error here: https://github.com/ScopeLift/umbra-protocol/pull/528/commits/8cc3c59e4f6901de1f6a8029e4f94ebb632128d9

Reply:
It seems we still have the new `nativeToken` variable and it's not clear to me why, would be great if we can remove that

Reply:
Sounds good, i'll work on that next

Reply:
Removed `nativeToken` here: https://github.com/ScopeLift/umbra-protocol/pull/528/commits/4e9622b4008cd0783bf0ba3ac96c380bf8658cb6